controllers/kubeletconfig_controller.go

```diff
@@ -38,6 +38,9 @@ import (
 
 	mcov1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
 
+	"github.com/pkg/errors"
```

Comment:
no please, let's use the standard library's `errors` package.

Reply:
Old code. We do

Reply:
It would have been better to not even introduce (=amend the first commit) rather than add and remove, but I can turn a blind eye

Reply:
Haste is from the devil